plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfiguration.java

```diff
@@ -14,14 +14,30 @@
 import com.google.inject.name.Named;
 
 import org.eclipse.che.api.core.util.SystemInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.eclipse.che.plugin.docker.client.DockerConnector.BOOT2DOCKER_CERT_PATH_PROPERTY;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.BOOT2DOCKER_HOST_PROPERTY;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.BOOT2DOCKER_TLS_VERIFY_PROPERTY;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.DEFAULT_BOOT2DOCKER_CERTS_DIR;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.DEFAULT_BOOT2DOCKER_URI;
+import static org.eclipse.che.plugin.docker.client.DockerConnector.UNIX_SOCKET_URI;
 
 /**
  * @author Alexander Garagatyi
  */
 public class DockerConnectorConfiguration {
+
+    /**
+     * Logger.
```

Comment:
OK, I prefer to leave it as this. Just habit to have everything commented on top of method and inside methods describing logic